flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/DateTimeUtils.java

```diff
@@ -385,6 +388,18 @@ public static TimestampData toTimestampData(int v, int precision) {
         }
     }
 
```

Comment:
To address this comment, I replaced 3 with a constant. 
After giving it some thought, I believe it's better to keep these functions in DateTimeUtils.java rather than moving them to toTimestampLtzFunction, since toTimestampLtzFunction already depends on DateTimeUtils.java as a library. What are your thoughts on this? @snuyanzin 

> from any user outside of TO_TIMESTAMP_LTZ 3 here is a magic number.
> I would suggest to have such calls only in your toTimestampLtzFunction class and don't put them here

Reply:
if you move them to toTimestampLtzFunction you can encapsulate the logic like mentioned in a comment https://github.com/apache/flink/pull/25763#discussion_r1890023415

Reply:
Moved this to the toTimestampLtzFunction.